dashboard/models.py

```diff
@@ -356,17 +369,4 @@ def __str__(self):
 
     class Meta:
         db_table = 'user'
-        unique_together = (('id', 'course_id'),)
-
-
-class ResourceAccess(models.Model):
-    id = models.AutoField(primary_key=True, verbose_name="Table Id")
-    resource_id = models.CharField(blank=True, max_length=255, null=False, verbose_name='Resource Id')
-    user_id = models.BigIntegerField(blank=True, null=False, verbose_name='User Id')
-    access_time = models.DateTimeField(verbose_name="Access Time")
-
-    def __str__(self):
-        return f"Resource {self.resource_id} accessed by {self.user_id}"
-
-    class Meta:
-        db_table = 'resource_access'
+        unique_together = (('id', 'course_id'),)
```

Comment:
Nit: This isn't related to what you're doing, but I think `unique_together` will be deprecated in the near future.

https://docs.djangoproject.com/en/3.0/ref/models/options/#unique-together

Reply:
It looks like this was just a notice for Django 3 and UniqueConstraint is only in Django 3. 

#952 is still a work in progress. I feel like this shouldn't wait on the update and we'd replace this a future Django update.

Reply:
I'm seeing this was introduced in Django 2.2, but yeah, it doesn't need to be resolved now.

https://docs.djangoproject.com/en/2.2/ref/models/options/#unique-together